specification/datasets/cost_and_usage/columns/billedcost.md

```diff
@@ -1,6 +1,17 @@
 # Billed Cost
 
-The [*billed cost*](#glossary:billed-cost) represents a [*charge*](#glossary:charge) serving as the basis for invoicing, inclusive of the impacts of all reduced rates and discounts while excluding the [*amortization*](#glossary:amortization) of relevant purchases (one-time or recurring) paid to cover future eligible *charges*. This cost is denominated in the [Billing Currency](#billingcurrency). The Billed Cost is commonly used to perform FinOps capabilities that require cash-basis accounting such as cost allocation, budgeting, and invoice reconciliation.
+Billed Cost represents the invoiced cost amount associated with a [*charge*](#glossary:charge) (cash-basis cost), inclusive of all applicable reduced unit prices and discounts, while excluding any portions of cost that are invoiced as part of related purchases (e.g., commitment purchases or marketplace offerings purchases) and attributed to cover this charge. Each charge is treated as a whole, with all applicable discounts applied uniformly to the entire *charge*. This cost is denominated in the [Billing Currency](#billingcurrency). Billed Cost is commonly used in FinOps activities that rely on cash-basis accounting, such as cost allocation, budgeting, invoice reconciliation, and reporting of actual invoiced amounts.
+
+From the perspective of cost coverage, a *charge* may be:
+
+* Independent: not covered by or designated to cover other *charges*; its total cost is both recognized and directly invoiced on the *charge* itself.
+* Covered by other *charges*: its cost is recognized on the *charge* itself, but may be fully or partially invoiced on other *charges*.
+* Designated to cover other eligible *charges*: its cost is not recognized on the *charge* itself, but is designated to offset the invoiced cost of other eligible *charges*.
+* Both: a *charge* may be simultaneously covered by other *charges* and designated to cover other eligible *charges*.
```

Comment:
You've listed 3 options and then refer to "Both" in the last, I'm not sure what this refers to.